File: ArryLoad_VB/ArryLoad/Declarations.vb
Imports System

Namespace ArryLoad
	Public Class Declarations
		Private Function IsCorrectDouble(str As String, <Out()> ByRef value As Double) As Boolean
			Dim flag As Boolean = Double.TryParse(str, value)
			Dim result As Boolean
			If flag Then
				Dim flag2 As Boolean = value >= 0.0
				result = flag2
			Else
				result = False
			End If
			Return result
		End Function
	End Class
End Namespace

File: ArryLoad/Declarations.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ArryLoad
{//abstract
    public  class Declarations
    {
        private bool IsCorrectDouble(string str, out double value)
        {
            bool result = true;

            if (double.TryParse(str, out value))
            {
                if (value >= 0)
                {
                    result = true;
                }
                else
                {
                    result = false;
                }
            }
            else
            {
                result = false;
            }

            return result;
        }
    }
}
